pkg/boot/kexec/memory_linux.go

```diff
@@ -186,7 +186,7 @@ func (rs Ranges) FindSpaceAbove(sz uint, minAddr uintptr) (space Range, err erro
 	return rs.FindSpaceIn(sz, RangeFromInterval(minAddr, MaxAddr))
 }
 
-// FindSpaceIn finds a continguous piece of sz points within Ranges and returns
+// FindSpaceIn finds a contiguous piece of sz points within Ranges and returns
```

Comment:
Are you sure that it should be "contiguous" and not "continuous". I'm not an expert, but seems Intersect returns a "continuous" or it can be both.
```go
// Intersect returns the continuous range of points common to r and r2 if there
// is one.
func (r Range) Intersect(r2 Range) *Range {
	if !r.Overlaps(r2) {
		return nil
	}
	i := RangeFromInterval(max(r.Start, r2.Start), min(r.End(), r2.End()))
	return &i
}
```

Also you forgot to sing commit.

Reply:
oh, wow, a typo that could be either word, or a new one, in a function I never quite figured out ;-)
I lean to continous myself but who knows? Maybe we need that new word.

Reply:
Fine, let's go with _continuous_ so people are not reminded of back-to-office memes all that much :-)

![contiguous office space](https://media3.giphy.com/media/l3fQg5KqDwny04gus/giphy.gif?cid=6c09b952el1juqokhkus4g1fwuuxi9hegh32fu0pk2x0jeds&ep=v1_internal_gif_by_id&rid=giphy.gif&ct=g)

Reply:
If we wanted to be more creative, I'd plead for _contignuous_ just because it would read like _continuous_ to some people

this word play stuff is contagious

Reply:
at a closer look, I found 3 more occurrences, _and_ there is indeed that word _contiguous_ in another comment above, now have fun tossing a :coin:  :-)